src/libAtomVM/bif.c

```diff
@@ -497,6 +497,17 @@ term bif_erlang_add_2(Context *ctx, uint32_t fail_label, int live, term arg1, te
     }
 }
 
+term bif_erlang_add_1(Context *ctx, uint32_t fail_label, int live, term arg1)
```

Comment:
I'd rather call this plus_1 instead of add_1: we are not making an addition, it is just an unary plus symbol.